src/step.js

```diff
@@ -68,10 +71,11 @@ class Step {
         this._auditTrail.description = this.description
 
         ret = this._auditTrail.return = await _runFunction()
-        if (ret) return ret;
+        if (ret) return ret
 
         ret = this._auditTrail.return = await _runNestedSteps()
-        return ret;
+        if (this && this.type == "use case") this._auditTrail.elapsedTime = stopwatch.stop(this.description).time
```

Comment:
here you are creating a cyclic dependency... 
Use Case knows about Step (since it is a specialized Step) but Step shouldn't know about that Use Case exists

Reply:
I removed the circular dependency, and now the step does not "know" the use case.